skimage/draw/draw.py

```diff
@@ -693,3 +693,57 @@ def bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape=None):
            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=uint8)
     """
     return _bezier_curve(r0, c0, r1, c1, r2, c2, weight, shape)
+
+
+def rectangle(origin, extent, shape=None):
+    """Generate coordinates of pixels within an nd-rectangle.
+
+    Parameters
+    ----------
+    origin : tuple
+        origin of the rectangle, (page,row,column,channel)
+    extent : tuple
+        extent (num_pages,num_rows,num_cols,num_chanels..) of the rectangle.
```

Comment:
@soupault any thoughts on implementing this as a separate function? OpenCV has a boundingRect function. It might reduce dependency requirements for users if scikit-image had a similar function. 
http://docs.opencv.org/2.4/modules/imgproc/doc/structural_analysis_and_shape_descriptors.html?highlight=boundingrect#boundingrect